tests/test_services.py

```diff
@@ -0,0 +1,66 @@
+import json
+from nose import with_setup
+from troposphere import Template
+from tropohelper.services import create_kinesis_stream, create_json_redshift_firehose_from_stream
+
+class test_stack(object):
+    """Test stack."""
+    def __init__(self):
+        """Intitialize our test stack."""
+        self.stack = Template()
+        self.env = "test"
+
+
+class TestServices:
+    """Test Kinesis services."""
+
+    def setup(self):
+        """Create our test environment."""
+        self.stack = test_stack()
+
+    def test_create_json_redshift_firehose_from_stream(self):
+        """Test creating vpc parameter."""
```

Comment:
doesn't seem to be an accurate doc block.

Reply:
fixed

Reply:
Maybe its github cache, but i still see """Test creating vpc parameter.""" which isn't what your function does.